pages.fr/common/gem.md

```diff
@@ -0,0 +1,36 @@
+# gem
+
+> A package manager for the Ruby programming language.
```

Comment:
Thanks for your contribution, this line seems to be untranslated.

Reply:
You're right, it totally is, thanks for spotting it.